app/views/hyrax/collections/show.html.erb

```diff
@@ -114,6 +114,14 @@
     <button type="submit" form="collection-search" class='btn btn-lg btn-primary collection-landing-button col-sm-10'><i class='glyphicon glyphicon-search'></i> Search</button>
   <% end %>
 
+  <!-- Create render display for the Local Contexts Icon -->
+  <% if !@collection.local_contexts.blank? || !@collection.local_contexts.empty? %>
```

Comment:
`#blank?` should act the same as `#empty?` across empty arrays, empty strings and doesn't trip up on `nil`. You shouldn't need the call to `#empty?` unless I missed something

Reply:
Got it. I will go in and see through all the code and remove those extra `empty?` code, glad to know `blank?` does all of that, I thought it only handle `nil`